assets/javascripts/discourse/templates/components/tc-message.hbs

```diff
@@ -21,6 +27,12 @@
         {{#unless message.staged}}
           <div class="tc-msgactions-hover">
             <div class="tc-msgactions">
+              {{d-button
+                class="btn-flat react-btn"
+                action=(action "startReaction" ".react-btn" SHOW_LEFT)
```

Comment:
Feels a little unusual to pass `.react-btn`, both here, and in the `.chat-message-react-btn` below.

Reply:
those are selectors to know which button was pressed, while passing in the `SHOW_LEFT` arg. I think it makes the JS much nicer.